implementation/src/main/java/io/smallrye/mutiny/converters/uni/UniToMultiPublisher.java

```diff
@@ -4,16 +4,16 @@
 
 import java.util.concurrent.atomic.AtomicReference;
 
+import org.reactivestreams.Publisher;
 import org.reactivestreams.Subscriber;
 import org.reactivestreams.Subscription;
 
 import io.smallrye.mutiny.Uni;
-import io.smallrye.mutiny.operators.AbstractMulti;
 import io.smallrye.mutiny.operators.AbstractUni;
 import io.smallrye.mutiny.subscription.UniSubscriber;
 import io.smallrye.mutiny.subscription.UniSubscription;
 
-public final class UniToMultiPublisher<T> extends AbstractMulti<T> {
+public final class UniToMultiPublisher<T> implements Publisher<T> {
```

Comment:
What's the reason to not be a `Multi` directly?

Reply:
I thought it's safer to have the `Infrastructure` involved again